poap-xyz/poap-gallery
# Feature/plt 484 update welook logo

## Summary

As Welook, we would like POAP to display our updated logo in the gallery page so our brand is respected and our users have a smoother experience.

## Type of change


## Screenshots

![Captura de Pantalla 2022-07-18 a la(s) 12 48 09](https://user-images.githubusercontent.com/51130069/179551391-a8ddfc04-3a1f-4710-afb7-d1762568c0ea.png)

## Testing

1. Access to [POAP Gallery Page](https://poap.gallery/)
2. Select an event and click to him, to move to the details page
3. Go to the Collections section
4. In the column named "Collection", you should be to see the new logo of welook.

## Ticket

PLT-484(https://poap-devs.atlassian.net/browse/PLT-484)

## Checklist:
